doc/building.rst

```diff
@@ -83,7 +83,7 @@ To set up your build environment for development using HomeBrew_:
 
 3. Add Qt from brew to the path::
 
-    export PATH=/usr/local/Cellar/qt5/5.x.y/bin/qmake
+    export PATH=$PATH:/usr/local/Cellar/qt5/5.x.y/bin/qmake
```

Comment:
Nice catch, just to make sure I'd put the qt5 path first just in case a Qt4 is already in PATH. `/qmake` should probably also not be in PATH.

`export PATH=/usr/local/Cellar/qt5/5.x.y/bin:$PATH`